scanner/Makefile

```diff
@@ -76,6 +76,8 @@ tag:
 ## Images ##
 ############
 
+image-prefix := scanner-v4
```

Comment:
maybe make this `stackrox/scanner-v4` so the print statements match the image name. Also, not important, but `repository-prefix` may be more accurate? `image-prefix` makes me think of the full image name, which includes the registry

Reply:
That's fine. I think it's OK the way it is. Thanks.